roles/login-to-lago/tasks/main.yaml

```diff
@@ -0,0 +1,7 @@
+- name: Add master's fqdn to /etc/hosts
+  lineinfile:
+    path: /etc/hosts
+    line: "{{ hostvars[groups['masters'][0]]['ansible_default_ipv4']['address'] }} {{ hostvars[groups['masters'][0]]['ansible_fqdn'] }}"
+
+- name: Login to OpenShift
```

Comment:
This is not needed if do the login inside `provision.yaml`

Reply:
For the APB this is temporarily required.